src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -2560,11 +2583,19 @@ static void read_action_locked(grpc_exec_ctx *exec_ctx, void *tp,
   GPR_TIMER_END("reading_action_locked", 0);
 }
 
+static void schedule_bdp_ping_locked(grpc_exec_ctx *exec_ctx,
```

Comment:
Do we not need to REF the transport anymore?

Reply:
We ref prior to calling this the first time, and unref once the chain completes... I'll add a comment.